modules/zcbor/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config ZEPHYR_ZCBOR_MODULE
+	bool
+
+config ZCBOR
+	bool "zcbor library"
+	depends on !MINIMAL_LIBC
```

Comment:
We have to be able to use both minimal libc and newlib with zcbor. The fix here would be to add the required ranges in minimal libc.

Reply:
Removed